### ISSUE TITLE
Tidymodels citation

### ISSUE THREAD

**Issue by skadauke** [tidymodels/tidymodels#35]
Hi, I'm writing a paper and would like to cite the `tidymodels` set of packages. Currently, `citation("tidymodels")` yields the following:

```
To cite package ‘tidymodels’ in publications use:

  Max Kuhn and Hadley Wickham (2020). tidymodels: Easily Install and Load the
  'Tidymodels' Packages. R package version 0.1.0.
  https://CRAN.R-project.org/package=tidymodels

A BibTeX entry for LaTeX users is

  @Manual{,
    title = {tidymodels: Easily Install and Load the 'Tidymodels' Packages},
    author = {Max Kuhn and Hadley Wickham},
    year = {2020},
    note = {R package version 0.1.0},
    url = {https://CRAN.R-project.org/package=tidymodels},
  }
```

Since tidymodels.org is now up, would you want to update this to point to the tidymodels.org web page?

**Comment by topepo** [comment]
Yes. That would be great. Thanks!

**Comment by juliasilge** [comment]
Closed in #36
